modules/kaiju/kaiju/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: kaiju_kaiju
+description: Taxonomic classification of metagenomic sequence data using a protein reference database
+keywords:
+  - classify
+  - metagenomics
+  - fastq
+  - db
+tools:
+  - kaiju:
+      description: Fast and sensitive taxonomic classification for metagenomics
+      homepage: https://kaiju.binf.ku.dk/
+      documentation: https://github.com/bioinformatics-centre/kaiju/blob/master/README.md
+      tool_dev_url: https://github.com/bioinformatics-centre/kaiju
+      doi: "10.1038/ncomms11257"
+      licence: ["GNU GPL v3"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input fastq/fasta files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+      pattern: "*.{fastq,fastq.gz,fasta,fasta.gz}"
```

Comment:
Can this also accept .fq/.fa/.fna/.fas etc?

Reply:
Yes, it should work with any type of extension for the fastq and fasta formats.

Reply:
```suggestion
      pattern: "*.{fastq,fq,fasta,fa,fsa,fas,fna,fastq.gz,fq.gz,fasta.gz,fa.gz,fsa.gz,fas.gz,fna.gz}"
```

Reply:
I guess although this indeed looks slight overkill :laughing: But better to be explicit :+1:

Reply:
Alright! Then I'll commit the overkill version instead :laughing: